app/assistify/migrations/server/startup/migrations.js

```diff
@@ -39,11 +40,17 @@ Meteor.startup(() => {
 
 Meteor.startup(() => {
 	console.log('Migrating from threads to Discussions');
+
+	const isThread = { parentRoomId: { $exists:true } };
+	const convertToDiscussion = {
+		$rename: { parentRoomId: 'prid' },
+		$set: { t: 'p' },
```

Comment:
Is this type change ```p``` is intended?

Reply:
Yes. Since the parent room access is always an alternative to being allowed access the discussion itself, all discussions in RC are `p`